test/integration/users.test.js

```diff
@@ -1,7 +1,6 @@
 const chai = require("chai");
 const { expect } = chai;
 const chaiHttp = require("chai-http");
-
```

Comment:
Please restore these lines as removing them makes the code less readable

Reply:
Sure I'll make the changes.